<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LaneShowImageDlgDesign</class>
 <widget class="QDialog" name="LaneShowImageDlgDesign">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>432</width>
    <height>513</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Lane Image</string>
  </property>
  <widget class="QSplitter" name="splitter">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>470</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <widget class="QDialogButtonBox" name="button_box">
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="standardButtons">
     <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
    </property>
   </widget>
  </widget>
  <widget class="QTextEdit" name="textEditFeatureInfo">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>411</width>
     <height>251</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxSelectLink">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>221</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>411</width>
     <height>73</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QGraphicsView" name="graphicView_0"/>
    </item>
    <item row="0" column="1">
     <widget class="QGraphicsView" name="graphicView_1"/>
    </item>
    <item row="0" column="2">
     <widget class="QGraphicsView" name="graphicView_2"/>
    </item>
    <item row="0" column="3">
     <widget class="QGraphicsView" name="graphicView_3"/>
    </item>
    <item row="0" column="4">
     <widget class="QGraphicsView" name="graphicView_4"/>
    </item>
    <item row="0" column="5">
     <widget class="QGraphicsView" name="graphicView_5"/>
    </item>
    <item row="0" column="6">
     <widget class="QGraphicsView" name="graphicView_6"/>
    </item>
    <item row="0" column="7">
     <widget class="QGraphicsView" name="graphicView_7"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>411</width>
     <height>73</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QGraphicsView" name="graphicView_8"/>
    </item>
    <item row="0" column="1">
     <widget class="QGraphicsView" name="graphicView_9"/>
    </item>
    <item row="0" column="2">
     <widget class="QGraphicsView" name="graphicView_10"/>
    </item>
    <item row="0" column="3">
     <widget class="QGraphicsView" name="graphicView_11"/>
    </item>
    <item row="0" column="4">
     <widget class="QGraphicsView" name="graphicView_12"/>
    </item>
    <item row="0" column="5">
     <widget class="QGraphicsView" name="graphicView_13"/>
    </item>
    <item row="0" column="6">
     <widget class="QGraphicsView" name="graphicView_14"/>
    </item>
    <item row="0" column="7">
     <widget class="QGraphicsView" name="graphicView_15"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>button_box</sender>
   <signal>accepted()</signal>
   <receiver>LaneShowImageDlgDesign</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_box</sender>
   <signal>rejected()</signal>
   <receiver>LaneShowImageDlgDesign</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
